agent/engine/serviceconnect/manager_linux.go

```diff
@@ -402,3 +414,16 @@ func getECSAgentLogPathContainer() string {
 	}
 	return path.Dir(ecsLogFilePath)
 }
+
+// GetSupportedAppnetInterfaceVerToCapabilities returns service connect capabilities
+// supported by ECS Agent to register for a selected AppNet version.
+// Suppose we decide to register ecs.service-connect.v2 capability for new AppNet version (ex: 1.24.0.0),
+// now if ecs.service-connect.*v1* is continuously being supported by 1.24.0.0,
+// we will then register multiple capabilities.
+// {
+//    "v1": ["ecs.service-connect.v1"],
+//    "v2": ["ecs.service-connect.v1", "ecs.service-connect.v2"]
+// }
+func (agent *manager) GetSupportedAppnetInterfaceVerToCapabilities() []string {
+	return supportedAppnetInterfaceVerToCapability[agent.AppnetInterfaceVersion]
+}
```

Comment:
Nit - maybe rename to `GetCapabilitiesForAppnetInterfaceVersion`

Reply:
Done:)